src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj

```diff
@@ -25,25 +25,21 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Uno.WinUI" Version="4.4.13" />
+		<PackageReference Include="Uno.WinUI" Version="5.0.48" />
```

Comment:
Is this the lowest version that we can use?

Reply:
[5.0.19](https://www.nuget.org/packages/Uno.WinUI/5.0.19)

Reply:
The lowest we can use is 5.0.19.
Since we do not have projects using Uno 5, don't we want to use the latest stable without affecting projects?